src/briefcase/config.py

```diff
@@ -71,9 +71,14 @@ def make_class_name(formal_name):
         if unicodedata.category(ch) in xid_continue or ch in {"_"}
     )
 
+    # If the class name is an empty string, we want to return None to indicate
+    # indicate it is invalid
+    if class_name == "":
+        class_name = None
```

Comment:
Not sure this part is needed - "" and None both evaluate as False. If the issue is the `classname[0]` on L81, you can preface that if with `if class_name and ...`